notificationservice.cpp

```diff
@@ -46,6 +46,7 @@ public slots:
                     else if(mReader.name() == "ai") appIcon = mReader.readElementText();
                     else if(mReader.name() == "su") summary = mReader.readElementText();
                     else if(mReader.name() == "bo") body = mReader.readElementText();
+                    else if(mReader.name() == "vb") vibrate = mReader.readElementText();
```

Comment:
"Alert" or "Notice" seem to be slightly more appropriate. For example "No Alert", "Normal Alert", "Strong Alert" would be generic enough to cover our future needs and still be clear on what it does.